itest/rust/src/builtin_tests/containers/array_test.rs

```diff
@@ -452,6 +452,12 @@ fn untyped_array_try_from_typed() {
     node.free();
 }
 
+#[itest]
+fn array_should_format_with_display() {
+    assert_eq!("[1, 2, 3, 4]", format!("{}", array![1, 2, 3, 4]));
+    assert_eq!("[]", format!("{}", Array::<real>::new()));
+}
```

Comment:
Could you use `assert_eq!(actual, expected)` convention like existing tests? I.e. swap arguments.

Also don't hesitate to create some "actual" variables for readability.

Reply:
Sounds good